src/Servers/Kestrel/test/InMemory.FunctionalTests/ResponseTests.cs

```diff
@@ -1471,7 +1534,7 @@ public async Task HeadResponseBodyNotWrittenWithSyncWrite()
         await using (var server = new TestServer(async httpContext =>
         {
             httpContext.Response.ContentLength = 12;
-            await httpContext.Response.BodyWriter.WriteAsync(new Memory<byte>(Encoding.ASCII.GetBytes("hello, world"), 0, 12));
+            httpContext.Response.Body.Write(Encoding.ASCII.GetBytes("hello, world"), 0, 12);
```

Comment:
https://github.com/dotnet/aspnetcore/pull/7110 removed the sync write which this test and `HeadResponseBodyNotWrittenWithSyncWrite` above were explicitly testing.